webpack.config.js

```diff
@@ -139,16 +142,20 @@ module.exports = {
 			const { chunk } = data;
 			const { entryModule } = chunk;
 			const { rawRequest } = entryModule;
+			const { name } = chunk;
+			const isMinified = name.indexOf( '.min' ) !== -1;
+
+			const dirName = isMinified ? name.split( '.' )[ 0 ] : name;
 
 			/*
 			 * If the file being built is a Core Block's frontend file,
 			 * we build it in the block's directory.
 			 */
 			if ( rawRequest && rawRequest.includes( '/frontend.js' ) ) {
-				return `./build/block-library/blocks/[name]/frontend.js`;
+				return `./build/block-library/blocks/${ dirName }/${ name }.js`;
```

Comment:
As you can see, I'm interpolating a variable instead of passing the WebPack [name] placeholder [here](https://github.com/WordPress/gutenberg/pull/31729/files#diff-1fb26bc12ac780c7ad7325730ed09fc4c2c3d757c276c3dacc44bfe20faf166fR158), and here. I don't know if that could lead to unintended consequences.

It seems to work fine, but would love insights from people more experienced with WebPack (and the GB WebPack scripts) to chime in and let me know if this could be problematic and if there's a better way to accomplish this 🙇🏻